docs/source/installation/troubleshooting.rst

```diff
@@ -182,3 +182,115 @@ operating system's documentation to find out where you can get it and how you
 have to install it.
 
 If you are unable to solve your problem, check out the `dvisvgm FAQ <https://dvisvgm.de/FAQ/>`_.
+
+(Windows) ``Python is not recognized as an internal or external command, operable program or batch file.``
+----------------------------------------------------------------------------------------------------------
+
+1. Open ``System Properties`` (Right click ``Computer`` in the start menu, or use the keyboard shortcut ``Win``+``Pause``)
+2. Click ``Advanced system settings`` in the sidebar.
+3. Click ``Environment Variables``...
+4. Select ``PATH`` in the ``System variables`` section
+5. Click ``Edit``
+6. Add Python's path to the end of the list (the paths are separated by semicolons). For example:
+
+.. code-block:: bash
+
+  C:\Windows;C:\Windows\System32;C:\Python27
+
+.. note::
+
+  Your Python install location may be in a different location
+
+(This section is taken from `StackExchange <https://superuser.com/questions/143119/how-do-i-add-python-to-the-windows-path/143121#143121>`_).
```

Comment:
Please use a double underscore after the link target to make it an anonymous reference.

Personally, I'd rather include something along the lines of
```
To fix this, you need to add the Python executable to your ``PATH`` environment variable.
The following steps are due to `Michael Mrozek from a StackExchange answer <...>`__.
```
and put the statement directly after the section heading. But that's a matter of taste, to some degree.